libs/printing/src/render.tsx

```diff
@@ -109,13 +109,16 @@ export async function renderToPdf(
     // be in the PDF, which allows us to determine the necessary height to fit
     // the page to the content. viewport height here is irrelevant, but we have to
     // set something.
+    const viewportHeight =
+      // Viewport height can't be infinity
+      height === Infinity ? PAPER_DIMENSIONS.Letter.height : height;
```

Comment:
When is `height` set to `Infinity` and why? It seems weird that libs/printing decides internally to use letter-sized paper in that case.

Reply:
`height` is `Infinity` when printing with `LetterRoll` size paper. I'm not very familiar with our uses of letter roll paper but I wanted to preserve the behavior.